Taskfile.yml

```diff
@@ -220,8 +236,38 @@ tasks:
       - task: run-unit-tests
       # - task: run-acceptance-tests
 
+  test-release:
+    desc: Run integration tests using latest release of OpsLevel Terraform provider
+    requires:
+      vars: [OPSLEVEL_API_TOKEN]
+    cmds:
+      - rm -rf {{.PLUGIN_DIR}}
+      - task: terraform-clean-from
```

Comment:
why do we clean here but not in `test-unreleased`

Reply:
Actually, we should be cleaning out the old state files. I'm adding that to `test-unreleased`.
Good catch! 👍